{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/locale/config.yaml

```diff
@@ -0,0 +1,92 @@
+# Configuration for i18n workflow.
```

Comment:
The `i18n_service` expects this directory to be named `translations` (the support that Jill introduced):

https://github.com/edx/xblock-utils/blob/18fcd70ff8eed6af17d41b16cf66a8161c9fb51b/tests/unit/test_resources.py#L132-L148